### ISSUE TITLE
The Dispersy tracker stress test could be part of our tests

### ISSUE THREAD

**Issue by devos50** [Tribler/dispersy#542]
When refactoring the bootstrap tests, I noticed that we have a stress test that sends a bunch of introduction requests to a specific tracker. I think this test can be a valuable addition to our testing suite. I'm talking about this test: https://github.com/Tribler/dispersy/blob/devel/tests/test_bootstrap.py#L242. Currently, this test is disabled and requires some work to make it functional again.

If we can reuse this test, we could modify the code such that it starts a local tracker (instead of using one of our live trackers) and sends request to that tracker. We already have some code to start a tracker in the same file.

**Comment by qstokkink** [comment]
I agree with the reason for disabling it though (`The stress test is not actually a unittest`). I think this is more of a Gumby integration test than a Dispersy unit test. In fact, I believe the entirety of `test_bootstap.py` should be moved to Gumby.
